packages/smooth_app/lib/pages/smooth_bottom_navigation_bar.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/pages/history_page.dart';
+import 'package:smooth_app/pages/scan/scan_page.dart';
+import 'package:smooth_app/pages/user_preferences_page.dart';
+
+class _Page {
+  const _Page({required this.name, required this.icon, required this.body});
+  final String name;
+  final IconData icon;
+  final Widget body;
+}
+
+enum SmoothBottomNavigationTab {
+  Profile,
+  Scan,
+  History,
+}
+
+class SmoothBottomNavigationBar extends StatelessWidget {
+  const SmoothBottomNavigationBar({
+    this.tab = _defaultTab,
+  });
+
+  final SmoothBottomNavigationTab tab;
+
+  static const SmoothBottomNavigationTab _defaultTab =
+      SmoothBottomNavigationTab.Scan;
+
+  static const List<SmoothBottomNavigationTab> _tabs =
+      <SmoothBottomNavigationTab>[
+    SmoothBottomNavigationTab.Profile,
+    SmoothBottomNavigationTab.Scan,
+    SmoothBottomNavigationTab.History,
+  ];
+
+  static const Map<SmoothBottomNavigationTab, _Page> _pages =
+      <SmoothBottomNavigationTab, _Page>{
+    SmoothBottomNavigationTab.Profile: _Page(
+      name: 'Profile', // TODO(monsieurtanuki): translate
+      icon: Icons.account_circle,
+      body: UserPreferencesPage(),
+    ),
+    SmoothBottomNavigationTab.Scan: _Page(
+      name: 'Scan or Search',
+      icon: Icons.search,
+      body: ScanPage(),
+    ),
+    SmoothBottomNavigationTab.History: _Page(
+      name: 'History',
+      icon: Icons.history,
+      body: HistoryPage(),
+    ),
+  };
+
+  static Widget getCurrentPage({
```

Comment:
do we only need to expose this method outside? it's only used in `main.dart`, maybe we can call it `getDefaultPage`